kvrocks.conf

```diff
@@ -1052,5 +1052,10 @@ rocksdb.partition_filters yes
 # Default: 0
 rocksdb.max_compaction_bytes 0
 
+# RocksDB sideloading move_files
+# Move files to the database directory instead of copying them.
+# Default: yes
+rocksdb.sideloading.move_files yes
```

Comment:
It can be in command arguments instead of a config option.

Reply:
addessed via c9fd7bf9e16b865482895c6984d08d62ba5d4fc5